src/components/Dashboard/Dashboard.js

```diff
@@ -21,11 +21,10 @@ const Dashboard = () => {
 			<Router>
 				<BottomNav />
 				<Switch>
-					<Route path='/Profile' component={Profile} />
-					<Route path='/ProfileEdit' component={ProfileEdit} />
-					<Route path='/Friends' component={Friends} />
-					<Route path='/MatchPage' component={MatchPage} />
-					<Route path='/' component={Start} />
+					<Route path="/profile" component={Profile} />
+					<Route path="/friends" component={Friends} />
+					<Route path="/matchPage" component={MatchPage} />
+					<Route path="/" component={Start} />
```

Comment:
I have no strong feelings on this but: are we camel casing all our routes?  I am fine with that, I just want to be consistent!

Reply:
this either